sirepo/pkcli/job_agent.py

```diff
@@ -754,14 +754,11 @@ async def _sbatch_status(self):
             )
             return
         self._status = r.group()
-        if self._status in ('PENDING', 'CONFIGURING'):
+        if not self._start_ready.is_set():
```

Comment:
Was this a bug? Or is this a consequence of having a real slot q? I don't understand.

Reply:
https://github.com/radiasoft/sirepo/pull/3372/commits/3f3de31188f07f2daf8e5ac68d3dd746d2d8967a I've reverted back to the old behavior per the line you commented on. In addition, we now wait for _start_ready in a callback instead of in the agent dispatcher loop. Implementing waiting for transitioning to run was considerably easier than I thought. One of these days I won't be afraid of code:/